linter_exclusions.yml

```diff
@@ -2201,11 +2241,48 @@ networkfabric l3domain create:
     redistribute_static_routes:
       rule_exclusions:
       - option_length_too_long
+networkfabric l3domain update:
+  parameters:
+    aggregate_route_configuration:
+      rule_exclusions:
+      - option_length_too_long
+    connected_subnet_route_policy:
+      rule_exclusions:
+      - option_length_too_long
+    redistribute_connected_subnets:
+      rule_exclusions:
+      - option_length_too_long
+    redistribute_static_routes:
+      rule_exclusions:
+      - option_length_too_long
+  rule_exclusions:
+  - missing_command_test_coverage
 networkfabric internalnetwork create:
   parameters:
     static_route_configuration:
       rule_exclusions:
       - option_length_too_long
+networkfabric internalnetwork update:
+  parameters:
+    static_route_configuration:
+      rule_exclusions:
+      - option_length_too_long
+  rule_exclusions:
+  - missing_command_test_coverage
+networkfabric externalnetwork update:
+  rule_exclusions:
+  - missing_command_test_coverage
+networkfabric ipextendedcommunity create:
+  parameters:
+    ip_extended_community_rules:
+      rule_exclusions:
+      - option_length_too_long
+networkfabric npb list:
+  rule_exclusions:
+  - missing_command_test_coverage
+networkfabric npb show:
+  rule_exclusions:
+  - missing_command_test_coverage
```

Comment:
Please adding the missing command test coverage

Reply:
Currently we are blocking the PATCH calls from user. Once PATCH is implemented completely, we will then unblock it.